code/modules/reagents/oldchem/reagents/reagents_toxin.dm

```diff
@@ -26,6 +26,234 @@
 		return
 
 
+// Terror Spider, Green, Harmless Pheremone
+
+
+/datum/reagent/terror_green_toxin
```

Comment:
I'd move this all into its own folder, along with the chemical reaction, that goes with the away mission code.